### ISSUE TITLE
Quick reply support

### ISSUE THREAD

**Issue by makefunstuff** [oarrabi/EXFacebook-Messenger#4]
https://developers.facebook.com/docs/messenger-platform/send-api-reference/quick-replies in facebook messenger you can also specify optional quick reply.
I think it should be also added into facebook_messenger_models. 
Currently message structure can possibly ignore quick replies.

**Comment by nsomar** [comment]
Hello @strangeworks as with the other issue, if you would like to take this on that would be great.
Do you think you'd be open to handle this too?

**Comment by makefunstuff** [comment]
I want to use your library, as there no alternatives I would like to add support of this functionality aswell.

**Comment by makefunstuff** [comment]
```
== Compilation error on file lib/httpotion.ex ==
** (ArgumentError) Access is not a protocol
    (elixir) lib/protocol.ex:76: Protocol.assert_protocol!/2
    lib/httpotion.ex:292: (module)
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6

could not compile dependency :httpotion, "mix compile" failed. You can recompile this dependency with "mix deps.compile httpotion", update it with "mix deps.update httpotion" or clean it with "mix deps.clean httpotion"
```
might be unrelated but attempting to run tests for the project i'm constantly receiving compilation error for the httppotion library, did you have such problems?

**Comment by nsomar** [comment]
@strangeworks Yes, this was an issue with an outdated httpotion. I submitted a pr that fixes it https://github.com/oarrabi/facebook_messenger/pull/5

Looks like httpotion and newer elixirs dont work well together

**Comment by nsomar** [comment]
Closing this issue since its solved

**Comment by strangeworks** [comment]
Currently there is postback support implemented, quick reply requires additional data structure definitions. I plan to provide pr this week.

**Comment by aguxez** [comment]
Any updates on this feature yet? Quick replies would be very useful.
